api/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+ENV SERVER_HOST=0.0.0.0
+RUN apt-get update -y
+RUN apt-get install -y python-pip python-dev build-essential
+COPY . /app
```

Comment:
I would consider putting the python source into an "app" folder making this line `COPY app /app`. Less clutter in the root directory and you can have files/directories here that wont be part of the image.